Userland/Libraries/LibGUI/FilePicker.h

```diff
@@ -64,14 +65,21 @@ class FilePicker final
         size_t tray_item_index { 0 };
     };
 
+    static String history_file_path();
+    void load_history();
+    void save_history();
+
+    Vector<String> m_path_history;
+    NonnullRefPtr<GUI::ItemListModel<String>> m_path_history_model;
+
     RefPtr<MultiView> m_view;
     NonnullRefPtr<FileSystemModel> m_model;
     String m_selected_file;
 
     RefPtr<GUI::Label> m_error_label;
 
     RefPtr<TextBox> m_filename_textbox;
-    RefPtr<TextBox> m_location_textbox;
+    RefPtr<ComboBox> m_location_combo;
```

Comment:
It's a Combo*Box*, so this should be `m_location_combobox`

Reply:
it was. i've seen `m_screen_combo` in https://github.com/SerenityOS/serenity/blob/72ba072768dd2e2a53b184f10b2555f3a4d2644b/Userland/Applications/DisplaySettings/MonitorSettingsWidget.cpp#L77

Reply:
Existing code violating our naming guidelines doesn't mean we should add more :)